string label
int32 rotation
---
